### ISSUE TITLE
MD013 sentence-per-line gets confused by inline code

### ISSUE THREAD

**Issue by tv42** [rvben/rumdl#545]
### rumdl version

0.1.58

### Installation method

Other

### Bug description

Punctuation inside `inline code` confuses the sentence splitter (but apparently only if there's space after it?)

### Steps to reproduce

```markdown
# Example

Rust macros look like `foo! {}` with the exclamation mark.
```

### Expected behavior

```console
printf '# Example\n\nRust macros look like `foo! {}` with the exclamation mark.\n'|rumdl check -
# (silence)
```

### Actual behavior

```console
printf '# Example\n\nRust macros look like `foo! {}` with the exclamation mark.\n'|rumdl check -
<stdin>:3:1: [MD013] Line contains 2 sentences (one sentence per line required) [*]
```

### Configuration

```toml
[MD013]
line-length = 0
reflow = true
reflow-mode = "sentence-per-line"
```

### Operating system

Linux

**Comment by tv42** [comment]
Oh, also, https://crates.io/crates/seams is a great library that understands English enough to avoid the "Dr." trap.

The docs aren't great, use it something like this:

```rust
    let detector = seams::sentence_detector::dialog_detector::SentenceDetectorDialog::new()
        .map_err(|e| format!("Failed to create sentence detector: {e}"))?;

    for paragraph in ... {
            let sentences = detector
                .detect_sentences_borrowed(text)
                .map_err(|e| format!("Failed to detect sentences: {e}"))?;
    }
```

**Comment by rvben** [comment]
Fixed in v0.1.61. Sentence splitting now ignores punctuation inside inline code spans, so backtick-wrapped content no longer confuses the [sentence-per-line](https://rumdl.dev/md013) check. Thanks for reporting, @tv42!
